plugins/social_network/social_network.cpp

```diff
@@ -253,7 +253,7 @@ namespace golos { namespace plugins { namespace social_network {
         uint32_t vote_limit
     ) const {
         std::vector<discussion> result;
-#ifndef IS_LOW_MEM
```

Comment:
Why do you remove this `ifndef IS_LOW_MEM`?

Reply:
@afalaleev Why do not remove it? Does it stores any memory, or any another cause to don't include it to LOW_MEM (when it depends on parent_author, and we enabled it)?